### DISCUSSION TITLE
1.9.4 "Make OPENING the default device_class for binary_sensor STATE", unpraktisch für Statuskanäle von IP-Schaltaktoren

### DISCUSSION THREAD

**Discussion by Baxxy13** [SukramJ/aiohomematic#514] [Q&A] [answered]
Seit der oben erwähnten Änderung werden auch alle Statuskanäle von IP-Schaltaktoren als Sensor-Klasse "Opening" angelegt.
Das sieht zum einen ziemlich komisch aus und ist zum anderen auch logisch falsch denn ein EINgeschalteter Statuskanal wird aktuell als "offen" angezeigt.
![HA_PCBS2_Opening](https://user-images.githubusercontent.com/54898249/177393373-ad22232a-b56f-40d4-a35f-bbfa232eb2fd.JPG)
Das sieht dann auch in Lovelace "komisch" aus:
![HA_PCBS2_Opening_2](https://user-images.githubusercontent.com/54898249/177416238-e5c897bc-f553-4dd0-9d1f-35804dc265fe.JPG)

Mit 1.9.0 passte das wesentlich besser:
![HA_PCBS2_Status_190](https://user-images.githubusercontent.com/54898249/177410247-44aab44a-7479-4a8a-bee3-32263c00a41c.JPG)

Für die als Kontakt konfigurierten Kanäle des MOD-RC8 passt "Opening" sehr gut, für die Statuskanäle der IP-Schaltaktoren leider nicht.
Vielleicht lässt sich das irgendwie "optimieren".

**Comment by SukramJ** [comment, marked as answer]
Wie man es macht ...
Ich werde die Änderung wieder zurücknehmen.
Eine Sonderbehandlung auf Kanalebene werde ich nicht einbauen, denn das ist mir zu viel Pflegeaufwand.

**Reply by Baxxy13** [reply]
Alles klar. 
Da vermutlich mehr IP-Schaltaktoren als "Kontakt-Eingangskanäle" genutzt werden macht das Sinn.
Die "Kontakt-Eingangskanäle" sollten sich ja auch mittels "customize" auf die eigenen Bedürfnisse anpassen lassen.
